db/seeds.rb

```diff
@@ -0,0 +1,8 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+Message.create([{ greeting: 'Good Morning' }, { greeting: 'Good day' }, { greeting: 'Good evening' }, { greeting: 'Good afternoon' }, { greeting: 'Good night' }])
```

Comment:
- [ ] [OPTIONAL] Great job done @Mylo16 , however, let's refactor it because it's too long.

``` ruby
greetings = ['Good Morning', 'Good day', 'Good evening', 'Good afternoon', 'Good night']
Message.create(greetings.map { |greeting| { greeting: greeting } })
```

I have only made it shorter using a loop and an array of greetings.

<img width="210" alt="Screenshot 2023-08-24 at 22 20 25" src="https://github.com/Mylo16/my_new_rails_app/assets/58692590/7f3da040-46b0-4302-80be-7eb68d9e3d9b">